run/testing/grpc_server_streaming.e2e_test.go

```diff
@@ -71,12 +72,19 @@ func TestGRPCServerStreamingService(t *testing.T) {
 	}
 
 	recvMsgs := 0
+	recvFailures := 0
 	for {
 		_, err := resp.Recv()
 		if err == io.EOF {
 			break
 		} else if err != nil {
-			t.Fatalf("rpc StreamTime.Recv: %v", err)
+			recvFailures++
+			if recvFailures < 5 {
```

Comment:
Nit: move `5` to a predefined constant? ([Context](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants))